jekyll/_cci2/workflows.md

```diff
@@ -49,7 +66,7 @@ workflows:
 
 ## Sequential Job Execution Example
 
-The following example shows a workflow with four sequential jobs. The jobs run according to configured  requirements, each job waiting to start until the required job finishes successfully as illustrated in the diagram. 
+The following example shows a workflow with four sequential jobs. The jobs run according to configured requirements, each job waiting to start until the required job finishes successfully as illustrated in the diagram. 
```

Comment:
Suggested edit: "The following example shows a workflow with four sequential jobs. Each job can declare which other jobs are required to be completed successfully before that job runs. Specifying multiple jobs under `requires` will wait for all of those jobs to complete successfully before running."